seed/models/building_file.py

```diff
@@ -378,11 +396,10 @@ def process(self, organization_id, cycle, property_view=None):
 
             # set the property_state to the new one
             property_state = merged_state
-        elif not property_view:
+        elif not property_view and promote_property_state:
             property_view = property_state.promote(cycle)
         else:
-            # invalid arguments, must pass both or neither
-            return False, None, None, "Invalid arguments passed to BuildingFile.process()"
```

Comment:
I don't understand how this else would ever trigger. Do `PropertyViews` evaluate weird?

Reply:
probably why I deleted it